ledger/src/snapshot_utils.rs

```diff
@@ -37,14 +37,20 @@ pub const TAR_VERSION_FILE: &str = "version";
 const MAX_SNAPSHOT_DATA_FILE_SIZE: u64 = 32 * 1024 * 1024 * 1024; // 32 GiB
 const VERSION_STRING_V1_1_0: &str = "1.1.0";
 const VERSION_STRING_V1_2_0: &str = "1.2.0";
-const OUTPUT_SNAPSHOT_VERSION: SnapshotVersion = SnapshotVersion::V1_2_0;
+const DEFAULT_SNAPSHOT_VERSION: SnapshotVersion = SnapshotVersion::V1_2_0;
```

Comment:
nits: I think this `const` definition can now go away now. It's only used by `impl Default` for `SnapshotVersion`. Maybe just fold this into there? Named as `DEFAULT_...` here, this reads a bit redundant.

Reply:
This is supposed to indicate the default output snapshot version - as new formats are added, they won't initially be made default until later - having everything together at the top should hopefully make this more apparent and easier to maintain.

Happy to remove it if you still feel it should go.